modules/migration-restic-timeout.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-restic-timeout_{context}']
+= Restic timeout
+
+If Restic times out, the following error appears in the Velero log:
+
+----
+level=error msg="Error backing up item" backup=velero/monitoring error="timed out waiting for all PodVolumeBackups to complete" error.file="/go/src/github.com/heptio/velero/pkg/restic/backupper.go:165" error.function="github.com/heptio/velero/pkg/restic.(*backupper).BackupPodVolumes" group=v1
+----
+
+The default value of `restic_timeout` is one hour (`1h`). You can increase this for large migrations, keeping in mind that a higher value may delay the return of error messages.
+
+.Procedure
+
+. In the web console, navigate to *Catalog* -> *Installed Operators*.
```

Comment:
Can you be clear if this is the CAM web console or the {product-title} web console?